### DISCUSSION TITLE
硕士毕业论文如何使单双页的页眉不同

### DISCUSSION THREAD

**Discussion by LilyXiehn** [TheNetAdmin/zjuthesis#195] [General]
目前硕士毕业论文的页眉是左边“硕士学位论文”右边章节标题
![1631509796(1)](https://user-images.githubusercontent.com/31917485/133027218-18ff927c-7ee8-4069-960c-91223b36cf5e.png)
怎样改为和本科生毕业论文一样单数页是章节标题双数页是“硕士学位论文”并且居中显示呢？
谢谢

**Comment by TheNetAdmin** [comment]
这个页眉是[研究生院的毕业论文规范](https://github.com/TheNetAdmin/zjuthesis-std/blob/master/standard/graduate/general/2016-%E6%B5%99%E6%B1%9F%E5%A4%A7%E5%AD%A6%E7%A0%94%E7%A9%B6%E7%94%9F%E5%AD%A6%E4%BD%8D%E8%AE%BA%E6%96%87%E7%BC%96%E5%86%99%E8%A7%84%E5%88%99.pdf)里制定的 (2.2.2.3 页码、页眉编写规则)

如要修改，可修改这部分代码

https://github.com/TheNetAdmin/zjuthesis/blob/79bbf3e4589e4d3a367513eef8b3e730f646aa73/config/format/general/layout.tex#L20

将 undergraduate 的代码复制到下面 graduate 部分即可

**Reply by LilyXiehn** [reply]
好的谢谢
